src/main/resources/swagger/cromwell.yaml

```diff
@@ -604,6 +604,9 @@ definitions:
       stderr:
         type: string
         description: Path to the standard error file for this call
+      backendLogs:
+        type: object
```

Comment:
Ha ok, yes although it will depend on the backend (it's an optional value).
Local backend doesn't have anything like this for example.
Jes has 3 (log, stdout and stderr), but another backend could only have one or whatever.
I can include an example in the case of Jes though.